frontend_tests/puppeteer_tests/drafts.ts

```diff
@@ -8,8 +8,16 @@ async function wait_for_drafts_to_disappear(page: Page): Promise<void> {
     await page.waitForSelector("#draft_overlay.show", {hidden: true});
 }
 
-async function wait_for_drafts_to_appear(page: Page): Promise<void> {
+async function wait_for_drafts_to_appear(
+    page: Page,
+    contains_draft: Boolean = false,
+): Promise<void> {
     await page.waitForSelector("#draft_overlay.show");
+    if (contains_draft) {
+        await page.waitForFunction(() =>
+            document?.activeElement?.classList?.contains("draft-info-box"),
```

Comment:
I don’t see what this has to do with the animation. Nothing waits for the animation before focusing the element either.

Reply:
Ohh Okay! 
So when we open drafts modal the first draft element is focused already?  
In the CI it worked 100 times, because of the delay of `waitForFunction`?